pkg/gadgets/internal/socketenricher/bpf/extension.bpf.c

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: (GPL-2.0 WITH Linux-syscall-note) OR Apache-2.0
+/* Copyright (c) 2022 The Inspektor Gadget authors */
+
+#include <linux/bpf.h>
+#include <linux/if_ether.h>
+#include <linux/ip.h>
+#include <linux/in.h>
+#include <linux/tcp.h>
+#include <linux/udp.h>
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_endian.h>
+
+// current_netns is automatically set to the correct value in userspace with
+// RewriteConstants()
+const volatile __u64 current_netns = 0;
+
+#include "sockets-map.h"
+
+#define L4_OFF (ETH_HLEN + sizeof(struct iphdr))
+
+unsigned long long load_byte(void *skb,
+			     unsigned long long off) asm("llvm.bpf.load.byte");
+unsigned long long load_half(void *skb,
+			     unsigned long long off) asm("llvm.bpf.load.half");
+unsigned long long load_word(void *skb,
+			     unsigned long long off) asm("llvm.bpf.load.word");
```

Comment:
Building on @mauriciovasquezbernal 's  2cf287a, they are now defined in only one place. It might not be the best location though. Could this be done in another PR?